grafana/data_source_library_panel_test.go

```diff
@@ -0,0 +1,44 @@
+package grafana
+
+import (
+	"testing"
+
+	gapi "github.com/grafana/grafana-api-golang-client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDatasourceLibraryPanel(t *testing.T) {
+	CheckOSSTestsEnabled(t)
+	CheckOSSTestsSemver(t, ">=8.0.0")
+
+	var panel gapi.LibraryPanel
+	// var dashboard gapi.Dashboard
+	checks := []resource.TestCheckFunc{
+		testAccLibraryPanelCheckExists("grafana_library_panel.test", &panel),
+		// testAccDashboardCheckExists("grafana_dashboard.test", &dashboard),
```

Comment:
Please remove this line if it will be commented out.

Reply:
https://github.com/grafana/terraform-provider-grafana/pull/370/commits/038acea6f902be6d643957d3940d691cba8f33a0